src/server/node_manager.c

```diff
@@ -4171,19 +4167,13 @@ update2_to_vnode(vnal_t *pvnal, int new, mominfo_t *pmom, int *madenew, int from
 	}
 
 	if (vn_resc_added > 0) {
-		int pyrc;
 		log_event(PBSEVENT_DEBUG2, PBS_EVENTCLASS_HOOK,
 			LOG_INFO, "update2_to_vnode",
 			"Restarting Python interpreter as resourcedef file has changed.");
 		pbs_python_ext_shutdown_interpreter(&svr_interp_data);
-		pyrc = pbs_python_ext_start_interpreter(&svr_interp_data);
-		if (pyrc != 0) {
+		if (pbs_python_ext_start_interpreter(&svr_interp_data) != 0) {
 			log_err(PBSE_INTERNAL, __func__, "Failed to restart Python interpreter");
-			/*
-			 * FIXME:
-			 * Should we return an error?
-			 * If so, does anything need to be undone?
-			 */
+			return PBSE_PYTHON_UNINITIALIZED;
```

Comment:
Rather than adding a new error type, I think we can just use PBSE_INTERNAL since that's what we log it as.

Reply:
Understood.